site/app/templates/submission/homework/CurrentVersionBox.twig

```diff
@@ -74,8 +74,13 @@
                     </p>
                 {% elseif active_version > 0 %}
                     <p class="red-message">
-                        Note: This version of your assignment will not be graded the instructor/TAs. <br />
-                        Click the button "Grade This Version" if you would like to specify that this version of your homework should be graded.
+                        {% if after_ta_open %}
+                            Warning: TA grading has already begun and your grade will automatically be set to zero if the active <br />
```

Comment:
It's not particularly relevant to the other changes in this PR but would it be possible to change the `<br>` here and online 81 of this file?  It will look different on different screen sizes and won't wrap dynamically.